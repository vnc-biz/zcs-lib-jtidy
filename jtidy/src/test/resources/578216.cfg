# Tidy configuration file for bug 578216
indent: yes
tidy-mark: false
indent-spaces: 0

indent= yes
tidy-mark= false
indent-spaces= 0
