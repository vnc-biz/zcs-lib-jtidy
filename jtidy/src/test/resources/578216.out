<!-- use "-indent" on command line -->
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
<title>
[ 578216 ] Incorrect indent of &lt;SPAN&gt; elements
</title>
</head>
<body>
<div class="indent">
<span class="jobsectionhead">Benefits:</span> <span class=
"jobsectiontext">Using a new Project Profile Knowledge
Base...</span><br>
<span class="jobsectionhead">Solutions:</span> <span class=
"jobsectiontext">Comprehensive intranet-based knowledge base
containing...</span><br>
<span class="jobsectionhead">Roles:</span> <span class=
"jobsectiontext">Drove site and content management
architecture...</span><br>
<span class="jobsectionhead">Technology:</span> <span class=
"jobsectiontext">Visual InterDev, IIS, ...</span>
</div>
</body>
</html>
